packages/mds-stream/types.ts

```diff
@@ -29,3 +29,9 @@ export type StreamItemType = string
 export type StreamItemData = string
 export type StreamItem = [StreamItemID, [StreamItemType, StreamItemData]]
 export type ReadStreamResult = [Stream, StreamItem[]]
+
+export interface WriteStream<T> {
```

Comment:
Come to think of it -- I think you can just use the `StreamProducer` type!

Reply:
yeet